fhir/resource_structure_dev/research_study.json

```diff
@@ -0,0 +1,111 @@
+{
+  "resourceType" : "ResearchStudy",
+  "id" : "example-study-ccpuk",
+  "url" : "",
+  "name" : "CCPUK",
+  "title" : "ISARIC/WHO Clinical Characterisation Protocol UK",
+  "label" : [{
+    "type": {
+      "text": "Study Code"
+    },
+    "value": "CVCCPUK"
```

Comment:
may require the data element 'partOf' to indicate a study is part of a larger study (e.g. for covid CVBCFGF is part of CVVCORE)